test/e2e/framework/framework.go

```diff
@@ -635,6 +635,11 @@ func KubeDescribe(text string, body func()) bool {
 	return ginkgo.Describe("[k8s.io] "+text, body)
 }
 
+// SIGDescribe is a wrapper function for ginkgo describe. Adds SIG namespacing.
+func SIGDescribe(sig, text string, body func()) bool {
```

Comment:
it would be great to have consistency between this PR, https://github.com/kubernetes/kubernetes/blob/master/test/e2e/node/framework.go  and https://github.com/kubernetes/kubernetes/pull/99016. I'd vote for named methods like `SIGCloudProviderDescribe`

Reply:
/hold

@SergeyKanzhelev : sure, waiting on https://github.com/kubernetes/kubernetes/pull/98957#issuecomment-779665867 for the pattern we'd like to follow and will update the PR accordingly.

Reply:
Thanks @navidshaikh, trying to get consensus here https://github.com/kubernetes/kubernetes/issues/98326#issuecomment-782744493